main.go

```diff
@@ -1,3 +1,19 @@
+/*
+
```

Comment:
It's missing the copyright line again. Isn't the header generated?

Reply:
Not for main.go, adding it.